packages/ses/src/scope-handler.js

```diff
@@ -67,7 +67,7 @@ export function createScopeHandler(
         // fall through
       }
 
-      // Properties of the global.
+      // Properties of the endowments.
       if (prop in endowments) {
```

Comment:
Are these localObjects now? The word `endowments` here is now very confusing/

Reply:
Please do fix. I need to understand this, and this terminology confusion adds too much to my other confusions.